src/lib/battery/battery.h

```diff
@@ -204,11 +206,18 @@ class Battery : public ModuleParams
 	uint8_t _warning{battery_status_s::BATTERY_WARNING_NONE};
 	hrt_abstime _last_timestamp{0};
 
-	float first_run_time;
-	bool first_run{true};
-	float cell_voltage_filtered;
-	float cell_voltage_filtered_load;
+	float cell_voltage_filtered{0};
+	float cell_voltage_filtered_load{0};
 	float soc_initial{0};
-	float sees_warning_last;
+	float _discharged_mah_initial{0};
+	const int lookup_size = 11;
```

Comment:
Picky point - but can you prepend all members with `_` (so `_lookup_size` etc.) so we keep to PX4 convention?  

Also they seem to use lower_case, so if we move SOCLookup here it should become _soc_lookup, and below you should use _oc_voltage, and _soc inside struct Lookup.